packages/graphql/src/translate/where/create-connection-where-and-params.ts

```diff
@@ -38,11 +38,11 @@ export default function createConnectionWhereAndParams({
     relationship: Relationship;
     relationshipVariable: string;
     parameterPrefix: string;
-}): [string, any] {
+}): [string, string, any] {
```

Comment:
This result signature is confusing. Using `[string, Record<string, any>]` is a relatively common and well-known signature (although not ideal) in our project as it is used widely to refer to Cypher+Params.
However, by adding a 3º element now the signature is increasingly confusing, as the types do not tell you what that middle parameter is

For this case, we should return an object instead, something like:

```suggestion
}): { cypher: string, subquery: string,  params:Record<string, any> } {
```